### ISSUE TITLE
The bash profile path to use in zsh

### ISSUE THREAD

**Issue by dankonino** [gatsbyjs/gatsby#6672]
Hi I would love to ask what can I link to my .zsh profile please.
Thanks :)

**Comment by m-allanson** [comment]
Hey @dankonino I don't think I understand your question.... could you provide some more info?

**Comment by dankonino** [comment]
What is the path that I should link because I am not using `.bash_profile` in terminal but rather oh `ohmyzsh` and it won't pick up the path that installation sets in bash but I need to specify it for zsh

**Comment by m-allanson** [comment]
Which path do you want to link? Which installation do you mean? Are you talking about the `gatsby` command? Gatsby is installed using `npm` and the `-g` flag. So the path to `gatsby` will be wherever `npm` installs global dependencies to. You can use `npm list -g` to see where that is for your machine.

Does that answer the question? If not, can you rephrase it as if I know nothing? 😄

**Comment by dankonino** [comment]
Hmm, thanks for this yeah you are actually right.
Seemsl like it was linked
```├── autopublish@0.0.8
├── create-react-app@1.5.2
├── gatsby-cli@1.1.58
├── npm@6.2.0
├── npm-cli-login@0.0.10
├── selenium-standalone@6.14.0
└── selenium-webdriver@4.0.0-alpha.1```

But the problem is I can run `autopublish` but can't do `gatsby`. I wonder what's the problem with this. Okay it just started working :D, but I didn't do anything different. Don't know why the path was not refreshed or problematic.. thanks :)

**Comment by m-allanson** [comment]
Huh, strange. Glad it's working now!
